codegen/test_data/endpoints/bar_bar_method_argwithheaders.gogen

```diff
@@ -31,15 +31,13 @@ import (
 
 	"github.com/opentracing/opentracing-go"
 	"github.com/pkg/errors"
+	module "github.com/uber/zanzibar/.tmp_gen/endpoints/bar/module"
+	workflow "github.com/uber/zanzibar/.tmp_gen/endpoints/bar/workflow"
+	endpointsBarBar "github.com/uber/zanzibar/examples/example-gateway/build/gen-code/endpoints/bar/bar"
```

Comment:
why latest imports still format like this? I thought it is reverted https://github.com/golang/tools/commit/a2dc47679d30b6c496245bafc6a166b46c5fe318

Reply:
I just did a glide up. I think this should be the most recent goimports.